### PR TITLE
Added vulnerability status

### PR DESCRIPTION
**Pull request by Cheshiriks**
- added vulnerability status
- added link to user from comment
- added new language for vulnerability

### CODE REVIEW

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt [hunk 1]
```diff
@@ -43,31 +44,31 @@ class VulnerabilityService(
 ) {
     /**
      * @param vulnerabilityIdentifier identifier of vulnerability
-     * @param isActive admin vulnerability check flag
+     * @param status status of vulnerability
      * @return vulnerability by name
      */
-    fun findByVulnerabilityIdentifierAndIsActive(vulnerabilityIdentifier: String, isActive: Boolean) = vulnerabilityRepository.findByVulnerabilityIdentifierAndIsActive(
+    fun findByVulnerabilityIdentifierAndStatus(vulnerabilityIdentifier: String, status: VulnerabilityStatus) = vulnerabilityRepository.findByVulnerabilityIdentifierAndStatus(
```
Comment (sanyavertolet) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt:50]: Suggestion to implement `VulnerabilityFilter` in order to escape many different methods with long names

Comment (Cheshiriks) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt:50]: This is already a big enough refactoring task. Let's leave it for another pr.

Comment (sanyavertolet) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt:50]: Ok but create an issue

**Review by nulls — APPROVED** [review]

**Review by sanyavertolet — APPROVED** [review]
Please do not forget to open an issue and refactor the code soon (no backlog)

File: save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityLanguage.kt [hunk 2]
```diff
@@ -13,10 +13,14 @@ import kotlinx.serialization.Serializable
...
     C("C"),
     CPP("Cpp"),
+    CSHARP("CSharp"),
     JAVA("Java"),
     JS("JS"),
     KOTLIN("Kotlin"),
     OTHER("Other"),
+    PHP("Php"),
+    PYTHON("Python"),
+    SQL("Sql"),
```
Comment (sanyavertolet) [inline save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityLanguage.kt:23]:
```suggestion
    C("C"),
    CPP("C++"),
    CSHARP("C#"),
    GO("Go"),
    JAVA("Java"),
    JS("JavaScript"),
    KOTLIN("Kotlin"),
    OTHER("Other"),
    PHP("PHP"),
    PYTHON("Python"),
    RUST("Rust"),
    SQL("Sql"),
    TS("TypeScript"),
```

**Review by sanyavertolet — APPROVED** [review]

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt:50]: Created

**Pull request merged**